docs/user/alternatives.rst

```diff
@@ -115,5 +115,6 @@ These are download metrics for the project. Note that these numbers can be artif
 .. _segment-geospatial: https://github.com/opengeos/segment-geospatial
 .. _TerraTorch: https://github.com/IBM/terratorch
 .. _scikit-eo: https://github.com/yotarazona/scikit-eo
+.. _geo-bench: https://github.com/ServiceNow/geo-bench
```

Comment:
Naming: do we want to call this geo-bench (git), geobench (pip), or GEO-Bench (paper)?

Reply:
geo-bench is fine